lib/generators/hyrax/templates/db/migrate/20200617002029_change_sipity_entity_specific_responsibility.rb.erb

```diff
@@ -0,0 +1,11 @@
+class ChangeSipityEntitySpecificResponsibility < ActiveRecord::Migration<%= migration_version %>
+  def change
+    if defined?(Sipity::EntitySpecificResponsibility)
```

Comment:
what's up with these conditionals? is there a more generic way of accomplishing this? 
`Sipity::EntitySpecificResponsibility.connect.adapter_name == "PostgreSQL"` seems particularly weird here.

will the up/down features of this `change` work, or is there a risk they'll break if the conditionals evaluate differently?

Reply:
I removed one of the guard clauses, relying instead on the `ActiveRecord::Base.connection` and assuming that the class would use the same connection.